plugins/wm-actions/wm-actions.cpp

```diff
@@ -267,6 +268,36 @@ class wayfire_wm_actions_t : public wf::plugin_interface_t
         return true;
     };
 
+    wf::activator_callback on_send_to_back = [=] (auto ev) -> bool
+    {
+        return execute_for_selected_view(ev.source, [] (wayfire_view view)
+        {
+            auto ws    = view->get_output()->workspace->get_current_workspace();
+            auto views =
+                view->get_output()->workspace->get_views_on_workspace(ws,
+                    wf::LAYER_WORKSPACE);
+            wayfire_view bottom_view = views[views.size() - 1];
+            if (view != bottom_view)
+            {
+                // Move view to bottom by stacking it below the bottom-most view
+                view->get_output()->workspace->restack_below(view, bottom_view);
+
+                // Change focus to the last focused view on this workspace
+                views =
+                    view->get_output()->workspace->get_views_on_workspace(ws,
+                        wf::LAYER_WORKSPACE);
+                view->get_output()->focus_view(views[0], false);
+            }
+
+            // Alert any relevant plugins that stack order has changed
```

Comment:
workspace-manager should emit this signal when restacking the view below, right? Why is this needed again here?

Reply:
This was to make it work better with follow-focus so the view the cursor is pointing to gets the focus. We have to do this after the focus command of course. However, I'm not sure if we actually want to do the follow-focus change I submitted any more, after thinking about it more. So we possibly should take this signal out for now.

Reply:
Removed the above change.